accept.go

```diff
@@ -12,11 +16,14 @@ agreement here:
 https://www.chef.io/online-master-agreement
 
 =========================================================================
-EOF
+`
 
-  # TO INFINITY AND BEYOND!
-  while true
-  do
-    sleep 5
-  done
-fi
+func main() {
+	if len(os.Args) > 1 && (os.Args[1] == "true" || os.Args[1] == "1") {
+		os.Exit(0)
+	}
+	fmt.Print(licenseMsg)
+	for {
```

Comment:
❓ Is this preferable over `for {}`?

Reply:
I think the idea was to not peg the CPU.  Without the sleep:

`root      1702 96.2  0.0 102656  1368 pts/0    Rl   13:17   0:24  \_ /hab/pkgs/chef/mlsa/1.0.1/20181115131739/bin/accept
`

Reply:
FWIW, I thought we could be clever and do a blocking read from a channel, but the race detector panic's us in that case.

Reply:
Oh, I was assuming that something was clever enough to deal with `for {}` and _not peg CPU_. Well, this of course makes perfect sense, thanks.

Reply:
💭 Could we `wg.Add(1); wg.Wait()`  or something? well, it's fine. nevermind.